.github/workflows/test-opencast-build.yml

```diff
@@ -73,6 +73,17 @@ jobs:
             -Dmaven.wagon.http.pool=false \
             -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
 
+        #This has to happen in a separate step or else various service classes aren't visible
+      - name: build javadocs
+        run: |
+          mvn javadoc:javadoc -Pnone \
```

Comment:
Do you want to add `javadoc:aggregate` to the run too? Because our developer documentation states:

> The commands `./mvnw clean install`, `./mvnw javadoc:javadoc javadoc:aggregate`, and `./mvnw site` should all succeed

See: https://docs.opencast.org/develop/developer/#participate/development-process/#reviews

Reply:
Sure, works for me.  The main goal here was to ensure the javadocs worked and I figured the aggregation would always work if the per-module builds work.